vm/systemSmartContracts/validator_test.go

```diff
@@ -472,7 +469,7 @@ func TestStakingValidatorSC_ExecuteStakeTooManyNodes(t *testing.T) {
 	arguments.Arguments = [][]byte{big.NewInt(3).Bytes(), key1, []byte("msg1"), key2, []byte("msg2"), key3, []byte("msg3")}
 
 	errCode := stakingValidatorSc.Execute(arguments)
-	assert.Equal(t, vmcommon.UserError, errCode)
+	assert.Equal(t, vmcommon.Ok, errCode)
```

Comment:
I'd suggest we also check that the log entry is present or test `stakeNodes` returns error when the log is present.